Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
       <TargetFramework>netcoreapp3.1</TargetFramework>
-      <Version>1.0.6</Version>
+      <Version>1.1.0</Version>
```

Comment:
@akshelton Would you mind adding a `PackageReleaseNotes` as well and mention the changes?

Reply:
@sameera Thank you for reminding me. I will add a section for `PackageReleaseNotes`.